examples/README.md

```diff
@@ -93,19 +95,26 @@ If you add the new video (e.g, examples/data/people-detection.mp4) to the existi
 $ python3 examples/classification_sample_video.py --input=examples/data/people-detection.mp4
 ```
 
+
 ## Python implementation for object detection
 
 For this example, we assume that you've already:
 
-* Installed TensorFlow on your system
+* Installed TensorFlow on your system (TF2.x)
 * Installed **Intel<sup>®</sup> </sup> OpenVINO<sup>TM</sup> integration with Tensorflow** on your system
 
+[**click-here**](https://github.com/openvinotoolkit/openvino_tensorflow/examples/TF_1_x/README.md) For the conversion of yolov3 darknet model using Tensorflow 1.15 and it's python object detection sample.
+
```

Comment:
pls modify "it's python object detection sample" to "object detection python sample to be used with this model"